src/Microsoft.Azure.SignalR.Common/Endpoints/ServiceEndpointManagerBase.cs

```diff
@@ -85,6 +89,9 @@ protected async virtual Task ReloadServiceEndpointsAsync(IEnumerable<ServiceEndp
 
                 UpdateEndpoints(endpoints, out var addedEndpoints, out var removedEndpoints);
 
+                _synchronizer.AddServiceEndpoints(addedEndpoints);
+                _synchronizer.RemoveServiceEndpoints(removedEndpoints);
```

Comment:
`UpdateEndpoints()` recognize Endpoint+EndpointType changes, but exclude AccessKey change. Should `synchronizer` values `AccessKey` change as well?